### ISSUE TITLE
[Programmatic Access – Make Code - Review, Commit and push local changes to GitHub] Ensures every id attribute value is unique

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3765]
User Experience:
User id attribute will not be defined for any control on the page then user who are visually impaired might affected while navigating on page using screen reader.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate through all the controls appearing.
7. Verify the issue

Actual Result:
Id attribute does not have unique value for the controls.

Expected Result:
Ensures every id attribute value is unique.

Issue: Ensures every id attribute value is unique (duplicate-id - https://dequeuniversity.com/rules/axe/4.0/duplicate-id?application=msftAI)

Target application:  Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#editor

Element path: #githubArea > .serialHeader.ui

Snippet: <div id="serialHeader" class="ui serialHeader">

How to fix:
Fix any of the following:
  Document has multiple static elements with the same id attribute: serialHeader

Environment: Microsoft Edge version 88.0.692.0

Note:
Same issue exists on Firefox & Chrome.

WCAG Reference:
[WCAG 4.1.1 – Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: Accessibility insights for web
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG4.1.1; #Win10-Edge(Chromium);

![WCAG4 1 1](https://user-images.githubusercontent.com/72871089/100225892-e7e8b000-2f44-11eb-8255-9f113fba8c8c.png)
